src/views/Viewer.vue

```diff
@@ -23,7 +23,7 @@
 
 <template>
 	<Modal
-		v-if="initiated || currentFile.modal"
+		v-show="initiated || currentFile.modal"
```

Comment:
No, this is on purpose.
What did you encountered that made you change this? :)

Reply:
It still a draft ! Don't look at my code 😳

On a more serious note, you're not gonna like it, the sidebar is remove by v-if before the closed event is dispatched, so even if the emit is called, the event listener is destroy and never called.

Reply:
Put the event listener in the entrypoint then?
https://github.com/nextcloud/viewer/blob/master/src/main.js

And pass the current state as a prop?

I'm unsubscribing, sorry :see_no_evil: